events/event.json

```diff
@@ -1,63 +1,63 @@
 {
```

Comment:
Don't modify event.json. Let this file remain as Is and create a new file `employee-create.json` and copy this content to the new file.

Reply:
I have created employee-create.json file and paste the content from event.json file.